packages/notifications/push/amplify_push_notifications_pinpoint/lib/src/push_notifications_background_processing.dart

```diff
@@ -19,14 +19,17 @@ const _backgroundChannel = MethodChannel(
 /// can record the notificaiton received event when the app is in a killed state.
 /// {@endtemplate}
 @pragma('vm:entry-point')
-Future<void> amplifyBackgroundProcessing() async {
+Future<void> amplifyBackgroundProcessing({
+  AmplifySecureStorage? amplifySecureStorage,
+}) async {
   WidgetsFlutterBinding.ensureInitialized();
-  final amplifySecureStorage = AmplifySecureStorage(
-    config: AmplifySecureStorageConfig(
-      scope: 'amplifyPushNotifications',
-    ),
-  );
-  final amplifyconfigStr = await amplifySecureStorage.read(
+  final amplifySecureStorage0 = amplifySecureStorage ??
```

Comment:
Setting the default value in the parameter list, so `amplifySecureStorage` can be non-nullable.